### DISCUSSION TITLE
What is the naming convention of version numbers in pdm?

### DISCUSSION THREAD

**Discussion by monchin** [pdm-project/pdm#2453] [Q&A]
Hi! I have a project with different hardwares, so I need to compile for both in one version. To seperate them, I'd like to name version as "0.1.0.a" and "0.1.0.b", but when I tried `pdm sync`, I found I failed with error `pdm.backend._vendor.packaging.version.InvalidVersion: Invalid version: '0.1.0.a+editable'`

I have seen other project with version number with letters, e.g. [pytz 2023.3.post1](https://pypi.org/project/pytz/2023.3.post1/), so I thought pdm could support it, but it seems not. So I'd like to know which naming standard pdm follows, and are there any suggestions to deal with this situation?

**Comment by monchin** [comment]
okay, I found that I can use `+`, e.g. `0.1.0+a`. But I still don't know why pytz could use version number like `2023.3.post1`.

**Reply by frostming** [reply]
Please read the specifications: https://packaging.python.org/en/latest/specifications/version-specifiers/
